programs/sunrise_beam/src/instructions/export_mint_authority.rs

```diff
@@ -0,0 +1,35 @@
+use anchor_lang::prelude::*;
+use anchor_spl::token::spl_token::instruction::AuthorityType;
+use anchor_spl::token::{set_authority, SetAuthority};
+
+use crate::{ExportMintAuthority, GSOL_AUTHORITY};
+
+pub fn handler(ctx: Context<ExportMintAuthority>) -> Result<()> {
```

Comment:
This looks good but could do with docs (top of the function line 6) and comments (inside)

Reply:
I'm not sure what the docs should indicate

Reply:
Basically each handler should explain what the instruction does. NBD though as I see you added docs in the lib module

Reply:
Yh. I think the comments are better placed in the root file, I'll work on making them more extensive though